### PR TITLE
Modified bomb and syndicate bomb spawner so Cuban Pete bombs maxcap and work again

### LINKED ISSUE #37259: The TTV won by beating an emagged arcade machine has an ineffective mixture
Title, beating Cuban Pete from an emagged arcade machine gives you a TTV bomb, however the mix used in the TTV no longer results in an explosion when the valve is opened.

Confirmed on Sybil - Round ID: 86605
Revision: 81e965bcf8a1b82e494d1d653cdbdc63136d575d

To reproduce: Be traitor, emag a battle arcade machine, outbomb Cuban Pete and win a TTV and Cuban pete hat, open the TTV valve, nothing happens other then the tanks rupturing after awhile.

Probably borked as a result of #37116

### PR DESCRIPTION
**Pull request by ohnopigeons**
:cl:
fix: The faint echoes of maracas grows louder, as if a past spirit once forgotten has come back with a vengeance...
/:cl:

I considered the solution of making a special TTV that only maxcaps and I didn't like the implications.

Fixes #37259

### CODE REVIEW

**Comment by AnturK** [comment]
Not a fan of mass produced maxcaps.

**Comment by Arianya** [comment]
Even you win the game consistently, TTV bombs still can't be put in backpacks and spawn in the open for all to see.

**Comment by Ratvar** [comment]
Arianya, you just make your own toy machines and play them forever in maint/spess untill you can delet station

**Comment by ExcessiveUseOfCobblestone** [comment]
> It just seems like a fruitless endeavor to have to constantly check this uncommon item every time atmos code gets changed to ensure the desired bomb effect is retained.

It doesn't need to be an actual TTV, especially since it's from a toy vending machine.

**Comment by Repukan** [comment]
If you're not a fan of mass produced maxcaps, where's the toxins removal PR then?

You can produce even more maxcaps in the time it'll take you to win two games.

**Comment by ExcessiveUseOfCobblestone** [comment]
The disgust of it being unique & some "spirit" of the Cuban Pete maymay (lol)  <<<<<<<<<< avoid having to maintain a hardly used feature anytime atmos code gets changed.

Unless you make it somehow dynamic to account for atmos changes so that it will forever do what it's intended to do, I really think this isn't the correct way to go about the item.

**Comment by pigeonsk** [comment]
I've included some amount of dynamicism, and these figures are optimized from plasma burn and temperature equations that are unlikely to change. Even then there's a large amount of tolerance as maxcap is lower than normal TTV capabilities by a fair margin. I can generalize it further to take into account the heat capacities of the gases while I still have the math on hand. All of this work is work that's already been done so it's less work for me than reasoning around a special TTV.

**Comment by ExcessiveUseOfCobblestone** [comment]
As long as it isn't going to have people go "oh no I forgot to check that one feature nearly no one uses!" (and actually cause issues with said feature) then that's fine.

**Comment by pigeonsk** [comment]
Plasma tank temperature is now fully generalized to account for or at least tolerate most possible changes, except for things like the removal of plasma fire.

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -1,10 +1,8 @@
 //Plasma fire properties
 #define OXYGEN_BURN_RATE_BASE				1.4
 #define PLASMA_BURN_RATE_DELTA				9
-#define PLASMA_UPPER_TEMPERATURE			(1370+T0C)
```
Comment (ShizCalev) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: hm

Comment (Arianya) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: That was moved to code/__DEFINES/atmospherics.dm, where PLASMA_MINIMUM_BURN_TEMPERATURE was already located.

Same for PLASMA_OXYGEN_FULLBURN

File: code/game/objects/effects/spawners/bombspawner.dm [hunk 2]
```diff
@@ -1,46 +1,66 @@
+#define CELCIUS_TO_KELVIN(T_K)	((T_K) + T0C)
```
Comment (cyclowns) [inline code/game/objects/effects/spawners/bombspawner.dm]: celsius

**Pull request merged**